app/models/concerns/to_csv.rb

```diff
@@ -8,31 +8,26 @@ def dashboard
       all
     end
 
+    # def column_names
```

Comment:
We were planning on including this to demonstrate to bespoke models that `dashboard` and `column_names` were required methods, can we achieve that and also document this required interface in a top-level comment.

Reply:
OK, something like this? [0c368b6](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/648/commits/0c368b6eb789a1cf6247a6bd7d09d4d90a2cfd9b)